test/fixtures/parser/README.md

```diff
@@ -20,8 +20,8 @@ within that yaml file.
 For best test coverage, add both a `.sql` and `.yml` file. The easiest way to
 add a `.yml` file is to run:
 
-    sqlfluff parse -f yaml test.sql | grep -v newline | grep -v whitespace | sed -e '1,2d' -e 's/^....//' > test.yml
-  
+    sqlfluff parse -f yaml 'ansi/empty_file.sql' | grep -v newline | grep -v whitespace | sed -e '1,2d' -e 's/^....//' > 'ansi/empty_file.sql'

```

Comment:
As I mentioned earlier, I recently a better way to do this. (The `sed` command in particular was fragile, not handling strings that span multiple lines.)

Reply:
Oops I didn't mean to commit this change. Would he great to have a built-in cli command to generate these. Mine if I add the gist as a command unless you want to?

Reply:
It would make sense that the logic to generate the yaml structure already exists in the code which asserts it though. We can hopefully just re use that

Reply:
I'll let you do it, if that's okay.

Yes, I had guessed there was similar code already in the test. I probably should've found that, rather than reinventing the wheel.

Reply:
@NiallRees - I'm also open to having an option in the `parse` command to explicitly output a file in the right format. 

I currently run `parse` on the file with the output set to `yaml` and then just copy and paste the result.

Reply:
I decided to just write a script which would automatically regenerate all the yml files for all the sql parse fixtures. Should make modifications to parse logic far quicker to test.